keyboards/bear_65/info.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Please regenerate this file without the rotation - it currently looks like this in Configurator:
![image](https://user-images.githubusercontent.com/4781841/93859246-bc590780-fd00-11ea-8e5d-7d7f0a344899.png)
You can test this out by pressing Ctrl+Shift+I to load a local info.json.